tfhe-zk-pok/src/curve_api/bls12_446.rs

```diff
@@ -373,9 +454,9 @@ mod g2 {
         }
 
         pub fn double(self) -> Self {
-            Self {
-                inner: self.inner.double(),
-            }
+            let mut this = self;
+            this.inner.double_in_place();
+            this
```

Comment:
yes, but i prefer this approach since the `mut` appears in the lsp signature helpers even though it's irrelevant, and the temporary binding doesn't cost us anything

Reply:
ok